Gordon360/Controllers/MembershipsController.cs

```diff
@@ -44,18 +43,11 @@ public ActionResult<IEnumerable<MembershipView>> GetMemberships(string? involvem
                 sessionCode: sessionCode,
                 participationTypes: participationTypes);
 
-            // When user is null, only SiteAdmin and Police can see all the user's memberships.
-            if ((username is null) && !(viewerGroups.Contains(AuthGroup.SiteAdmin)
-                || viewerGroups.Contains(AuthGroup.Police)))
-            {
-                memberships = _membershipService.RemovePrivateMemberships(memberships, authenticatedUserUsername);
-                return Ok(memberships);
-            }
             // Only user, siteAdmin and Police can see all the user's memberships.
-            else if (!(username == authenticatedUserUsername
-                || viewerGroups.Contains(AuthGroup.SiteAdmin)
-                || viewerGroups.Contains(AuthGroup.Police)
-                ))
+            if (
+              (username is null || username != authenticatedUserUsername)
+              && !(viewerGroups.Contains(AuthGroup.SiteAdmin) || viewerGroups.Contains(AuthGroup.Police))
```

Comment:
Confirming that all memberships will be shown if user is police or siteadmin.  If user is neither of those then the memberships will be shown only if the username is not null and matches the authenticated username.